synthetics-client/suites/sample_testsuite.yml

```diff
@@ -0,0 +1,45 @@
+---
+  name: "Sample Test Suite"
+  type: "http"
+  appId: ""
+  notebookId: ""
+  savedQueryId: ""
+  savedVisualizationId: ""
+  operationalPanelId: ""
+  enabled: true
+  ipv4: true
+  ipv6: true
+  resolverMode: "all"
+  timeoutSeconds: 16
+  tags:
+  - "news"
+  - "apis"
+  keep_null: true
+  hosts:
+  - "https://opensearch.org/"
+  - "https://opensearch.org/synthetics"
+  - "https://github.com/opensearch-project"
+  maxRedirects: 0
+  proxyURL: ""
+  proxyHeaders: {}
+  username: ""
+  password: ""
+  ssl: {}
+  indexHeaders: true
+  indexResponse: "always"
```

Comment:
There are many fields of the yml that not being used in the code. Remove them if it's unnecessary. You may also want to add comments to a sample test suites, so users will understand each field and it's meaning

Reply:
For example, what are the available units for a interval job